src/partials/template.html

```diff
@@ -1,20 +1,35 @@
 <div class="progress-list-panel">
   <div ng-if="!ctrl.isPanelAlert">
-    <div ng-repeat="item in items" ng-if="!ctrl.isMultibar">
-      <progress-list-plugin-progress
-        class="progress-list-item"
-        item="item"
-        on-hover="ctrl.onHover($index, event)"
-        on-mouse-leave="ctrl.onMouseLeave()"
-      />
-    </div>
-    <div ng-repeat="item in items" ng-if="ctrl.isMultibar">
-      <progress-list-plugin-multibar-progress
-        class="progress-list-item"
-        item="item"
-        on-hover="ctrl.onHover($index, event, title, value)"
-        on-mouse-leave="ctrl.onMouseLeave()"
-      />
+    <div ng-repeat="item in items" class="list-item progress-item">
+      <div class="list-item-title"
+        ng-bind-html="item.title"
+        style="margin-top: 10px; margin-bottom: -8px;"
+      ></div>
+      <div 
+        class="list-item-content" 
+        style="width: {{ item.aggregatedProgress }}%;"
+        ng-mousemove="ctrl.onHover({ index: $index, event: $event })"
+        ng-mouseleave="ctrl.onMouseLeave()"
+      >
+        <div
+          ng-repeat="percentValue in item.percentValues track by $index"
+          style="
+            width: {{ percentValue }}%;
+            display: inline-block;
+            background-color: {{ item.colors[$index] }};
+            opacity: {{ item.opacity }};
+            height: {{ item.titleParams.barHeight }}px;
+            margin-top: {{ item.titleParams.titleTopMargin }}px;
+          "
```

Comment:
Lets add `getItemStyle` method

Reply:
@VargBurz I like the idea! thank you

Reply:
but actually not: the thing is that you don't want to keep your logic of rendering in code as much as possible. 
It's MVP pattern here, right? So we better keep all view-related logic into view. What do you think?

Reply:
yeah, it makes sense.